src/Processors/Formats/Impl/CSVRowInputFormat.cpp

```diff
@@ -36,6 +36,86 @@ CSVRowInputFormat::CSVRowInputFormat(
             ErrorCodes::BAD_ARGUMENTS);
 }
 
+void CSVRowInputFormat::readPrefix()
```

Comment:
Please add a comment with explanation why we need this.

Reply:
Done.

Reply:
done